### DISCUSSION TITLE
AI算命，你信嗎?

### DISCUSSION THREAD

**Discussion by wenchi9023** [1101-GS4719-C/Python-Community-GS4719-C#848] [General]
https://www.techbang.com/posts/85389-what-is-the-principle-behind-ai-fortune-telling-in-china-right

市面上越來越多面相APP主打電腦工程師與專業面相大師合作，透過生物特徵和過往數據來評斷用戶面相。
雖然大家都知道將個資和照片上傳網站或應用軟體都會有極大的風險，但還是有人會去嘗試並且相信，頗為矛盾。好奇大家相信面相嗎? 會不會想嘗試呢?

**Comment by Chiway1229** [comment]
我個人對於算命是沒有什麼太大的興趣
不過只要跟ai結合了，就讓我想要試試

**Comment by eason8479** [comment]
我個人是相信選擇能改變命運，所以不太信算命的
還有就是要看他能說的多細，還是只是隨便拉個很大的範圍+一些很模糊的描述
(EX:30~40會有血光之災，那我在31流鼻血算不算)

還有，那個連結好像失效了，要不要再重貼看看

**Reply by wenchi9023** [reply]
感謝告知，已更新!

**Comment by Pietra1226** [comment]
如果單純論面相，我覺得AI說不定還比真人的準，畢竟資料量大，訓練也比較完善，但缺乏的就是人跟人互動之間的感覺，或許算命師有一部分是透過互動去了解一個人。

**Comment by Kotori0501** [comment]
我個人是什麼都會想嘗試一下，但不會完全相信
感覺AI用在算命上是很酷的一件事

**Comment by a45320614** [comment]
大部分AI看面相的app都是中國開發的，現在的技術也不成熟到能真的準確的算出一個人的命運，我個人認為應該要注意個人隱私方面的問題，不要隨意上傳個人臉部照片，避免成為巴納姆效應下的受害者

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
在我看來 AI算命比人為操作有更大的參考價值 畢竟它有更大的資料庫

**Comment by ching020721** [comment]
AI算命比較偏科學上的算命，聽系上教授說他曾經去算過，科學算命相對比較準確，因此我會覺得很新鮮想嘗試。

**Comment by StevenWanglolz** [comment]
我認為AI算命可能真的比人算的要準。AI是根據大批過往的數遽去對未來進行合理的推測，而人的算命是根據"神力"? 不知道大家有沒有聽過"拉普拉斯的惡魔"這個概念，在知道全宇宙的每一個粒子的位置以及運動模式後就可以完整地預測未來。這只是一個概念而已，因為現實世界是做不到的，但我認為這一概念代表的是未來是固定的?
歡迎討論哈哈哈

**Comment by ruthchen1020** [comment]
我覺得AI算命是算比較制式化的東西，像是生辰八字可以知道的東西，但很多人算命是想問事情，這種通常會根據個人目前的處境去看，我覺得AI可能沒辦法比老師傅還準確。
而且算命通常都有一點代價，不知道AI算命的話是誰扛。

**Comment by Zoe06060606** [comment]
我覺得AI算命就把它當作樂趣就好 不要太信以為真 此外 也要注意隱私的安全性 別隨便上傳自己的個人資料

**Comment by Evan0979** [comment]
算命我都不一定相信了,AI算命嗎......

**Comment by williamyen0623** [comment]
AI竟然還能算命，只能說人類的想像力真的很豐富，什麼都做得出來，真有趣

**Comment by manipulateed** [comment]
其實就是一個統計學的範疇，不是每個都符合甚至會是極端值，都只能當參考用。

**Comment by evanchou1204** [comment]
只能說信不信AI會算命的邏輯基點是建立在相信算命上的，算命的都說，一命二運三風水，四積陰德五讀書，命運這種東西就算提早知道也改變不了什麼，還不如好好讀書！

**Comment by CPCpyt** [comment]
我覺得能有參考價值，
我覺得甚至比神棍幫我算的還要靠普多了！

**Comment by Doreen331** [comment]
其實用面相算命也算是一種大數據分析吧，只是現在把這些大數據訓練成一個模型，取代那些面相算命師，而且訓練出來的結果說不定更準確

**Comment by Floriaand** [comment]
我會想嘗試耶，因為感覺很酷，不過我都是信好不信壞，就是只聽自己想聽的那種人。

**Comment by peggy0329** [comment]
透過大數據來分析每個人的命運、面相等，或許是會比一般占卜準
但我還是不會信，畢竟每個人都有各自的特色，同時我覺得後天努力也會改變命運

**Comment by dinosaur1009** [comment]
AI算命是一件結合統計還有各種科學所得出來的結果，相較於市面上算命，是科學的分析結果，但是算命這種事準不準，終究是見仁見智ㄌ

**Comment by Guagua1234** [comment]
我自己有曾經讓學校老師卜卦過，
那時的我身體超健康，但老師卻說要我好好注意身體，
因為卦上說我之後身體會非常不好，
直到我近期被送急診才開始相信，
所以我認為不管何種方法都可以進行嘗試，
但不要影響到身心狀況，導致生活重心偏移就好了，
還是要注意一下隱私問題啦

**Comment by Valentinaa0214** [comment]
我本身算是有點小迷信，
AI算命利用大量的數據和算法進行分析，從而給出相對準確的結果，
這跟一般算命其實沒有甚麼太大的不同，
只是相對理性一點，感覺是個有趣的小玩意兒，我會想試試看！！

**Comment by Sylvie0601** [comment]
我覺得超級酷!我其實不太相信跟看面相，我更相信平常相處去看一個人，未來的東西透過自己去努力爭取而得到，但我覺得跟AI結合在一起我也會想要試試看!

**Comment by ghost** [comment]
我覺得算命就是一種大數據的集合預估，所以加上ai來分析自然是一個可發展的方向，不過算命終歸只是基於數據的推測，如果數據量不足，或就剛好處在例外區，那自然無法成功預測

**Comment by Twenty0522** [comment]
如果在非常迷惘，理性思考無法決定時，可能會求神問卜，但保持半信半疑的態度，而使用AI會降低我對算命的相信程度，因為對我來說算命是玄學，AI是科學。

**Comment by yuxinlin93** [comment]
雖然我不太相信算命 但AI算的可信度會較高一點，畢竟是有資料庫，一堆樣本去計算出來的，但也只能當作參考，因為計算的還是命運這種充滿未知數，甚至不按邏輯走的東西，算命頂多當作大數據的建議，要不要採納還是看當事人如何抉擇

**Comment by s96015welson** [comment]
我覺得AI算命滿有發展力的，由於AI背後有大量的資料庫與數據，總結出來的結論是有根據的，因此有一定的參考性。不過若是把算命的定義範圍在放大些，按照興趣推送可能會感興趣的影片或廣告是否也算是一種AI算命呢?這樣感覺就有說服力多了。

**Comment by HsuChen88** [comment]
小時候臉書上的小測驗都會玩了，AI算命大家應該也會抱持著嘗試心態去算吧，如果不用錢，我可能會考慮。

**Comment by 110302550** [comment]
其實算命對我來說一直都是一個騙人的東西，不會特別去相信，因為我的人生由我做主，我命由我不由天。
